README.md

```diff
@@ -67,10 +66,43 @@ $ foreman s
 
 ### Change CMS URL Path
 
-In development, frontend will use the local CMS for convenience. You can change the MAS_CMS_URL on .env file. Use http://cms.moneyadviceservice.org.uk to point to LIVE content, https://cms.qa.dev.mas.local for testing Or http://localhost:PORT to point to a local running CMS.
+In development, frontend will use the local CMS for convenience. See [CMS repository README](https://github.com/moneyadviceservice/cms/blob/master/README.md) for instructions on setting up a local CMS instance.
+
+You can change the MAS_CMS_URL on .env file. Use https://cms.qa.dev.mas.local for testing, or http://localhost:PORT to point to a local running CMS.
 
 Don't forget to restart the server after the modification.
 
+### Development setup gotchas
+
+#### Contento 404 error
+
+```
+Unable to fetch Footer JSON from Contento error: [#<Core::Connection::Http::ResourceNotFound: the server responded with status 404>] url_prefix: [#<URI::HTTP http://localhost:3000/>]
+```
+
+The CMS is setup but the database is empty. A solution is to copy the QA CMS database into the CMS, as detailed in the [CMS repository README](https://github.com/moneyadviceservice/cms/blob/master/README.md).
+
+#### Problems loading Dough or Yeast
+
+Assuming you have run `bowndler install`, you may have issues with previous `bower` installations.
+
+```
+rm -rf vendor/assets/bower_components
```

Comment:
@paulanthonywilson you can do it with/without. I just want it comprehensively vanquished :)